### ISSUE TITLE
HTML render error when yield chatbot more than once

### ISSUE THREAD

**Issue by cycychenyi** [gradio-app/gradio#6061]
### Describe the bug

When I yield chatbot with plain text, it works well. When I yield chatbot once with HTML, it works well too. But when I yield chatbot with HTML more than once, it cannot work as expected. Big thanks in advance.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import gradio as gr
import mdtex2html
import time


def predict(user_input, chatbot):
    chatbot.append(())
    response = ''
    for i in range(2):
        chatbot[-1] = (user_input, mdtex2html.convert(user_input * (i + 1)))
        time.sleep(1)
        yield chatbot


with gr.Blocks() as demo:
    chatbot = gr.Chatbot()
    with gr.Row():
        with gr.Column(scale=4):
            with gr.Column(scale=12):
                user_input = gr.Textbox(show_label=False, placeholder="Input...", lines=10)
            with gr.Column(min_width=32, scale=1):
                submitBtn = gr.Button("Submit", variant="primary")

    submitBtn.click(predict, [user_input, chatbot], [chatbot], show_progress=True)

demo.queue().launch(server_name='0.0.0.0', share=False, inbrowser=True)
```

### Screenshot

![image](https://github.com/gradio-app/gradio/assets/48085050/6ec97f44-dd18-41cf-a307-7c1beaa28c9e)

### System Info

```shell
Gradio Environment Information:
------------------------------
Operating System: Linux
gradio version: 3.48.0
gradio_client version: 0.6.1

------------------------------------------------
gradio dependencies in your environment:

aiofiles: 23.2.1
altair: 5.1.2
fastapi: 0.103.2
ffmpy: 0.3.1
gradio-client==0.6.1 is not installed.
httpx: 0.25.0
huggingface-hub: 0.18.0
importlib-resources: 6.1.0
jinja2: 3.1.2
markupsafe: 2.1.3
matplotlib: 3.8.0
numpy: 1.26.0
orjson: 3.9.9
packaging: 23.2
pandas: 2.1.1
pillow: 10.0.1
pydantic: 2.4.2
pydub: 0.25.1
python-multipart: 0.0.6
pyyaml: 6.0.1
requests: 2.31.0
semantic-version: 2.10.0
typing-extensions: 4.8.0
uvicorn: 0.23.2
websockets: 11.0.3
authlib; extra == 'oauth' is not installed.
itsdangerous; extra == 'oauth' is not installed.


gradio_client dependencies in your environment:

fsspec: 2023.9.2
httpx: 0.25.0
huggingface-hub: 0.18.0
packaging: 23.2
requests: 2.31.0
typing-extensions: 4.8.0
websockets: 11.0.3
```

### Severity

Blocking usage of gradio

**Comment by abidlabs** [comment]
Looks like this is now fixed in the latest version of `gradio` (4.38.1)!

**Comment by dsantiago** [comment]
I'm also with this issue
